harness/determined/core/_checkpoint.py

```diff
@@ -260,6 +260,7 @@ def _upload_single(
         *,
         selector: Optional[Callable[[str], bool]] = None,
     ) -> str:
+        logger.info(f"Uploading checkpoint to {ckpt_dir} metadata={metadata}")
```

Comment:
I would keep all the new logs at the debug level.
Also, I think this log is incorrect: we are uploading `ckpt_dir` content to the storage (not checkpoint to `ckpt_dir`). Similar problem in the next log.